src/Pdf/Engine/WkHtmlToPdfEngine.php

```diff
@@ -93,6 +93,15 @@ protected function _getCommand()
             throw new Exception(sprintf('wkhtmltopdf binary is not found or not executable: %s', $this->_binary));
         }
 
+        $binaryWrapper = $this->config('binaryWrapper');
+
+        if ($binaryWrapper) {
+            if (!is_executable($binaryWrapper)) {
```

Comment:
How about we just remove that `is_excutable()` check? The [output()](https://github.com/FriendsOfCake/CakePdf/blob/master/src/Pdf/Engine/WkHtmlToPdfEngine.php#L38) method does check for error after the command is run and throws exception with the error message. So if someone used wrong binary name/path he would still info about what went wrong.